### ISSUE TITLE
[vendor:cnv-qe@redhat.com][level:component]DataVolume tests Verify DataVolume should [test_id:3932]succeed creating dv from imageio source

### ISSUE THREAD

**Issue by yuvalturg** [kubevirt/containerized-data-importer#1276]
/kind bug
/triage build-officer

https://prow.apps.ovirt.org/view/gcs/kubevirt-prow/pr-logs/pull/kubevirt_containerized-data-importer/1271/pull-containerized-data-importer-e2e-os-3.11.0-crio/1277724303965556736

https://prow.apps.ovirt.org/view/gcs/kubevirt-prow/pr-logs/pull/kubevirt_containerized-data-importer/1258/pull-containerized-data-importer-e2e-os-3.11.0-crio/1277612069230743553

```• Failure [344.691 seconds]
[vendor:cnv-qe@redhat.com][level:component]DataVolume tests
/go/src/kubevirt.io/containerized-data-importer/tests/datavolume_test.go:31
  Verify DataVolume
  /go/src/kubevirt.io/containerized-data-importer/tests/datavolume_test.go:69
    should
skipped 437 lines unfold_more
[Fail] [vendor:cnv-qe@redhat.com][level:component]DataVolume tests Verify DataVolume should [It] [test_id:3932]succeed creating dv from imageio source 
/go/src/kubevirt.io/containerized-data-importer/tests/datavolume_test.go:161
```

**Comment by maya-r** [comment]
Should be fixed by #1254

**Comment by awels** [comment]
It failed on the OS-3.11 lane, I will have a party once that lane is retired.
